Formula/juju.rb

```diff
@@ -1,9 +1,9 @@
 class Juju < Formula
   desc "DevOps management tool"
-  homepage "https://jujucharms.com/"
+  homepage "https://juju.is/"
   url "https://github.com/juju/juju.git",
-    tag:      "juju-2.8.2",
-    revision: "a44e6eb38430da695737f5e9f37819478b9587c3"
+    tag:      "juju-2.8.3",
+    revision: "ab69570b38fbc746e54184e4c3274612bcbb8327"
   license "AGPL-3.0"
```

Comment:
```suggestion
  license "AGPL-3.0-only"
```

License header:
```
// Copyright 2013 Canonical Ltd.
// Licensed under the AGPLv3, see LICENCE file for details.
```

Reply:
I have seen this couple of times. It does not quite necessarily mean they want to use AGPL-3.0-only (which I assume), but simply mean they did update their outdated license info.

Reply:
Thanks though.